website/tables/aws/aws_networkmanager_global_networks.md

```diff
@@ -0,0 +1,31 @@
+# Table: aws_networkmanager_global_networks
+
+This table shows data for Networkmanager Global Networks.
+
+https://docs.aws.amazon.com/networkmanager/latest/APIReference/API_GlobalNetwork.html
+The  'request_region' column is added to show region of where the request was made from.
```

Comment:
It doesn't seem like there is a `request_region` column on this table

Reply:
This has been resolved